packages/hoppscotch-common/src/components/environments/my/index.vue

```diff
@@ -32,19 +32,36 @@
       :environment="environment"
       @edit-environment="editEnvironment(index)"
     />
-    <HoppSmartPlaceholder
-      v-if="environments.length === 0"
-      :src="`/images/states/${colorMode.value}/blockchain.svg`"
-      :alt="`${t('empty.environments')}`"
-      :text="t('empty.environments')"
-    >
-      <HoppButtonSecondary
-        :label="`${t('add.new')}`"
-        filled
-        outline
-        @click="displayModalAdd(true)"
-      />
-    </HoppSmartPlaceholder>
+    <div v-if="environments.length === 0" class="p-5 space-y-25 flex flex-col">
+      <div class="flex flex-col items-center space-y-4">
+        <span class="text-secondaryLight">
+          {{ t("environment.import_or_create") }}
+        </span>
+        <div class="flex items-center space-x-4">
```

Comment:
When the width of the app is slightly smaller, the buttons begin to overflow sidewise. The second button should come underneath the first button in such cases

<img width="904" alt="image" src="https://github.com/hoppscotch/hoppscotch/assets/70131076/ab167384-cad3-400f-8b30-a4018ddaabdc">